test/datasets/test_metrla.py

```diff
@@ -0,0 +1,38 @@
+import shutil
+
+from torch_geometric.datasets import MetrLa
+
+
+def test_metrla() -> None:
+    root = '/tmp/metr_la'
```

Comment:
```suggestion
    root = osp.join('/tmp', 'metr_la')
```
As the tests suggests, this might need to be fixed on Windows, *e.g.*, via `osp.join('/tmp', 'metr_la')`.